src/snowflake/snowpark/dataframe_reader.py

```diff
@@ -1116,6 +1116,10 @@ def dbapi(
         custom_schema: Optional[Union[str, StructType]] = None,
         predicates: Optional[List[str]] = None,
         session_init_statement: Optional[Union[str, List[str]]] = None,
+        imports: Optional[List[str]] = None,
+        packages: Optional[List[str]] = None,
+        external_access_integration: Optional[str] = None,
+        use_udtf_ingestion: bool = False,
```

Comment:
I have some new idea, rather than calling it `use_udtf_ingestion`, what about having a param called ***_mode, and default is  parquet copy into mode, and another value is called UDTF mode.